core/sequencer/sequencer.go

```diff
@@ -192,13 +208,13 @@ func genEpochTicks(t util.TimeSource, last time.Time, minTick <-chan time.Time,
 
 // newMutations returns a list of mutations to process and highest sequence
 // number returned.
-func (s *Sequencer) newMutations(ctx context.Context, startSequence int64) ([]*tpb.EntryUpdate, int64, error) {
+func (s *Sequencer) newMutations(ctx context.Context, mapID int64, startSequence int64) ([]*tpb.EntryUpdate, int64, error) {
```

Comment:
You can write:  mapID, startSequence int64 here
Although these are both int64, you might take the view that these are really different types (since they represent different things) and not combine them.

Reply:
combining is fine